provider/boltmem/boltmem.go

```diff
@@ -30,38 +30,59 @@ import (
 var (
 	bktNotificationInfo = []byte("notification_info")
 	bktSilences         = []byte("silences")
-	bktAlerts           = []byte("alerts")
+	// bktAlerts           = []byte("alerts")
```

Comment:
I think this could be removed. If we want to keep the comment, we should probably at least add an explanation as to why it's commented out.